src/ui/Facet/Facet.ts

```diff
@@ -52,10 +52,10 @@ import { OmniboxValueElement } from './OmniboxValueElement';
 import { OmniboxValuesList } from './OmniboxValuesList';
 import { ValueElement } from './ValueElement';
 import { ValueElementRenderer } from './ValueElementRenderer';
-import { DependentFacetManager } from './DependentFacetManager';
 import { AccessibleButton } from '../../utils/AccessibleButton';
 import { IResponsiveComponentOptions } from '../ResponsiveComponents/ResponsiveComponentsManager';
 import { ResponsiveFacetOptions } from '../ResponsiveComponents/ResponsiveFacetOptions';
+import { DependsOnManagerUtils } from '../../utils/DependsOnManagerUtils';
```

Comment:
I think we could reduce the name slightly to just `DependsOnManager` without losing any of the meaning. No need for the `utils` in the filename or class name.